plugins/chat/spec/lib/chat/duplicate_message_validator_spec.rb

```diff
@@ -1,121 +1,108 @@
 # frozen_string_literal: true
 
 describe Chat::DuplicateMessageValidator do
-  let(:chat_channel) { Fabricate(:chat_channel) }
+  let(:message) { "goal!" }
+  fab!(:category_channel) { Fabricate(:chat_channel) }
+  fab!(:dm_channel) { Fabricate(:direct_message_channel) }
+  fab!(:user) { Fabricate(:user) }
```

Comment:
shouldn't this be caught by our rubocop rule?

Reply:
What should be caught? 🤔

Reply:
our shorthand:

```suggestion
  fab!(:user)
```